src/Domain.LinnApps/MaterialRequirements/MaterialRequirementsReportService.cs

```diff
@@ -146,9 +156,9 @@ public class MaterialRequirementsReportService : IMaterialRequirementsReportServ
             {
                 this.filterQuery = a => a.JobRef == jobRef && a.PreferredSupplierId == supplierId;
             }
-            else if (partSelector == "Stock Category")
+            else if (partSelector == "Stock Category Name")
             {
-                this.filterQuery = a => a.JobRef == jobRef && a.StockCategoryName == stockCategoryName;
+                this.filterQuery = a => a.JobRef == jobRef && a.StockCategoryName.ToUpper() == stockCategoryName.ToUpper();
```

Comment:
I hope C# gets EqualsIgnoresCase soon (checked and isn't there yet though, got confused with js I think)